atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -281,6 +285,8 @@ private enum State {
     private final ExpectationsMetrics expectationsDataCollectionMetrics;
     private volatile long cellCommitLocksRequested = 0L;
     private volatile long rowCommitLocksRequested = 0L;
+    private volatile long cellChangeMetadataSent = 0L;
```

Comment:
separate / non-actionable: We're slowly racking up the things this class is tracking - it might be worth refactoring the whole metrics around transactions stuff later on.